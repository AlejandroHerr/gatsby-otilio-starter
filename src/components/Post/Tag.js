// @flow
import React from 'react';
import Link from 'gatsby-link';

import styles from './Post.module.scss';

const Tag = ({ tag }: {tag: string}) => (
  <Link className={styles.post_tags__tag} to={`/tags/${tag.toLowerCase()}`}>{tag}</Link>
);

export default Tag;
