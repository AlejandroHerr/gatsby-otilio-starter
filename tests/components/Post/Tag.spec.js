import React from 'react';
import renderer from 'react-test-renderer';

import Tag from '../../../src/components/Post/Tag';

const defaultProps = {
  tag: 'test',
};

describe('Post', () => {
  describe('Tag', () => {
    it('should match the exact snapshot', () => {
      const tree = renderer.create(<Tag {...defaultProps} />).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
