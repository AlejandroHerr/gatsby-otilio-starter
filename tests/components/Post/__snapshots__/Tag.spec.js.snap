// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post Tag should match the exact snapshot 1`] = `
<a
  className="post_tags__tag"
  href="/tags/test"
>
  test
</a>
`;
